ui/app/templates/components/transit-key-action/decrypt.hbs

```diff
@@ -33,6 +36,27 @@
         </div>
       </div>
     {{/if}}
+    {{#if (or (eq @key.type "rsa-2048") (eq @key.type "rsa-3072") (eq @key.type "rsa-4096"))}}
+      <div class="field">
+        <label for="padding_scheme" class="is-label">Padding Scheme</label>
+        <div class="control is-expanded">
+          <div class="select is-fullwidth">
+            <select
+              name="padding_scheme"
+              id="padding_scheme"
+              data-test-padding-scheme="true"
+              onchange={{action (mut @padding_scheme) value="target.value"}}
+            >
+              {{#each (array "oaep" "pkcs1v15") as |scheme|}}
+                <option selected={{if @padding_scheme (eq @padding_scheme scheme) (eq scheme "oaep")}} value={{scheme}}>
```

Comment:
I noticed this conditional differs than the one above in `transit-key-action/datakey.hbs` - is that intentional? If we want to set `oaep` as the default for every `padding_scheme`, I recommend doing so in`transit-key-actions.js` instead, which makes the default consistent and also simplifies this to:
```suggestion
                <option selected={{eq @padding_scheme scheme}} value={{scheme}}>
```